Expensify/App
# [NO QA] Fix `tests/unit/getPullRequestsMergedBetweenTest.sh`

### Details
The [`-S` flag](https://git-scm.com/docs/git-cherry-pick#Documentation/git-cherry-pick.txt--Sltkeyidgt) was causing cherry picks to be signed, we can safely remove the flag and the tests will still be effective.

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/191675

### Tests
1. Turn off signing in your git config
2. Run the script locally and verify it passes 👍 
3. Verify the tests pass in this PR

### QA Steps
N/A - Unit test, no code changes
